tests/data/journal_article.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<collection xmlns="http://www.loc.gov/MARC21/slim">
+  <record>
+    <leader>     nas  2200301 a 4500</leader>
+    <controlfield tag="001">jnl1</controlfield>
+    <controlfield tag="008">231104c2023    xx  x goo     0   a|eng||</controlfield>
+    <datafield tag="024" ind1=" " ind2=" ">
+      <subfield code="2">test</subfield>
+      <subfield code="a">journal1</subfield>
```

Comment:
Should this be jnl1 rather than journal1? I'm not sure why the hierarchy ID would be different from the record ID in this context.

Reply:
It doesn't make a difference for this case, but I changed it for consistency.